EXILED/Exiled.Events/Commands/Hub/Install.cs

```diff
@@ -89,9 +89,11 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
                 releaseToDownload = foundRelease;
             }
 
-            Log.Info($"Downloading release \"{releaseToDownload.TagName}\". Found {releaseToDownload.Assets.Length} asset(s) to download.");
+            ReleaseAsset[] releaseAssets = releaseToDownload.Assets.Where(x => x.Name.IndexOf("-nwapi", StringComparison.OrdinalIgnoreCase) == -1).ToArray();
```

Comment:
Using `-nwapi` would in-theory detect: `someplugin-0.0.1-nwapi.dll` but not nwapi-someplugin.0.0.1.dll
So, if we're searching for anything with `nwapi` wouldn't that be better? Since if the resulting DLL is a NWAPI/LabAPI plugin, the string might apply to more cases than `-nwapi`.

This is only a small nitpick, I'm ok with the current solution.

```suggestion
            ReleaseAsset[] releaseAssets = releaseToDownload.Assets.Where(x => x.Name.IndexOf("nwapi", StringComparison.OrdinalIgnoreCase) == -1).ToArray();
```

Reply:
> Using `-nwapi` would in-theory detect: `someplugin-0.0.1-nwapi.dll` but not nwapi-someplugin.0.0.1.dll So, if we're searching for anything with `nwapi` wouldn't that be better? Since if the resulting DLL is a NWAPI/LabAPI plugin, the string might apply to more cases than `-nwapi`.
> 
> This is only a small nitpick, I'm ok with the current solution.

I agree